test/extensions/wasm/test_data/logging_rust/src/lib.rs

```diff
@@ -10,14 +10,14 @@ extern "C" {
 
 #[no_mangle]
 #[allow(non_snake_case)]
```

Comment:
You can remove `#[allow(non_snake_case)]` here and in other functions.

Reply:
done